src/main/java/com/onarandombox/MultiverseCore/api/MVDestination.java

```diff
@@ -1,142 +0,0 @@
-/******************************************************************************
- * Multiverse 2 Copyright (c) the Multiverse Team 2011.                       *
- * Multiverse 2 is licensed under the BSD License.                            *
- * For more information please check the README.md file included              *
- * with this project.                                                         *
- ******************************************************************************/
-
-package com.onarandombox.MultiverseCore.api;
-
-import org.bukkit.Location;
-import org.bukkit.entity.Entity;
-import org.bukkit.plugin.java.JavaPlugin;
-import org.bukkit.util.Vector;
-
-/**
- * A destination API for Multiverse
- * Any plugin can add these to MV and when they are, any action that uses them (portals, MVTP, etc.) can use them!
- */
-@Deprecated
-public interface MVDestination {
```

Comment:
Why is this being removed in this PR?

Reply:
I wanted to remove it in the destination revamp last time, but it was reference in old permission classes that was revamped in this PR. So now it can be removed.